### DISCUSSION TITLE
How is the rating calculated?

### DISCUSSION THREAD

**Discussion by breakTBB** [aropan/clist#314] [Q&A] [answered]
How is the rating standardized across different websites for the problem?

**Comment by breakTBB** [comment]
Why does this  AGC-018-b problem only have a rating of 1329? https://atcoder.jp/contests/agc018/tasks/agc018_b

**Reply by breakTBB** [reply]
The rating is at the same level as some medium LeetCode problems, which is unreasonable.

**Comment by aropan** [comment, marked as answer]
Rating not standardized. Problem rating is corresponding resource rating.
But if you have enough number of contests each resources then you could see luck value (probability to solve problem) - problems with same luck will be with "same rating" for you.

**Reply by breakTBB** [reply]
Thank you. I used to think that the ratings for questions on different websites were consistent when I filterd the problem by rating. it seems that I should actually filter them based on luck.

**Reply by ABarpanda** [reply]
Is there any mathematical formula based on which the rating of each question is calculated?
